src/Commands/AzureAD/RegisterAzureADApp.cs

```diff
@@ -94,6 +94,11 @@ public class RegisterAzureADApp : BasePSCmdlet, IDynamicParameters
 
         protected override void ProcessRecord()
         {
+            if (PSUtility.IsUserLocalAdmin())
+            {
+                WriteWarning("Running this cmdlet in requires you to run PowerShell as an administrator due to a regression in .NET 9.");
```

Comment:
Maybe use the word bug instead of regression? More (non dev) people will understand that better, I would think?